src/command.test.js

```diff
@@ -30,3 +30,30 @@ test(`is stateless`, () => {
   isWindowsMock.__mock.returnValue = true
   expect(commandConvert('$test')).toBe(commandConvert('$test'))
 })
+
+test(`converts embedded unix-style env variables usage for windows`, () => {
+  isWindowsMock.__mock.returnValue = true
+  expect(commandConvert('$test1/$test2/$test3')).toBe(
+    '%test1%/%test2%/%test3%',
+  )
+})
+
+test(`converts embedded windows-style env variables usage for linux`, () => {
+  isWindowsMock.__mock.returnValue = false
+  expect(commandConvert('%test1%/%test2%/%test3%')).toBe(
+    '$test1/$test2/$test3',
+  )
+})
+
+test(`
```

Comment:
I'm guessing you had to do this because of linting? How about instead you just make it one line and add:

```
// eslint-disable-next-line max-len
```

On the line above this one. Would that be ok?